normandy/base/api/permissions.py

```diff
@@ -19,3 +19,12 @@ class AdminEnabledOrReadOnly(AdminEnabled):
 
     def has_permission(self, request, view):
         return super().has_permission(request, view) or request.method in permissions.SAFE_METHODS
+
+
+class IsSuperUser(permissions.BasePermission):
```

Comment:
I know we kept talking about only super users having this permission, but I think we should probably use the normal Django permissions for the user object instead of a custom permission. Then we would not grant that permission to anyone. Super users would be able to use the API by virtue of automatically passing all permission checks.